src/components/ImportGithub.js

```diff
@@ -0,0 +1,60 @@
+import { Fragment, useState } from 'react'
+import { div, h, h2 } from 'react-hyperscript-helpers'
+import { ButtonPrimary } from 'src/components/common'
+import { TextInput } from 'src/components/input'
+import { submitMethod } from 'src/components/method-common'
+import { withBusyState } from 'src/libs/utils'
+
+// { tooltip: 'Link must start with https://github.com or https://raw.githubusercontent.com' },
+const ImportGithub = ({ setLoading, signal, onDismiss }) => {
+  const [methodName, setMethodName] = useState('')
+  const [methodVersionName, setMethodVersionName] = useState('')
+  const [methodUrl, setMethodUrl] = useState('')
+
+  return div({ style: { marginLeft: '4rem' } }, [
+    div({ style: { width: 500 } }, [h2(['Workflow Link'])]),
```

Comment:
TOL: Should we put red `*` or the orange warning `!` next to each field names to indicate that all 3 fields are required?

Reply:
If I disable the button until all three fields have entries, would it be too much to also have 3 warnings as well? Should we do one or the other?

Reply:
Yeah I think it is worth adding either the red `*` or orange `!` next to these fields so that user knows they are required (just like we do with inputs). And disabling the `Add` button until all 3 fields have values will ensure we don't allow sending empty values to API (where it will eventually fail)

Reply:
I was trying to find if we have the notion of `*` in Terra UI and it seems we do have a black colored `*`. And the create button is disabled with error stating why it is disabled. 
![Screenshot 2023-05-08 at 11 16 35 AM](https://user-images.githubusercontent.com/16748522/236862465-99882e26-ce06-4387-a1a0-d51f5deccec2.png)